src/vma/dev/ring.h

```diff
@@ -54,7 +54,8 @@ class ring_slave;
 
 typedef enum {
 	CQT_RX,
-	CQT_TX
+	CQT_TX,
+	CQT_RX_TX
```

Comment:
I dont like this new parameter hack.
Can we use ring_slave:m_type in order to decide which cq_type we should use ?

Reply:
:+1: